src/models/CronTabModel.ts

```diff
@@ -0,0 +1,25 @@
+import { modelToRef } from '@crontab-utils/utils';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
+export const CronTabModel: K8sModel = {
```

Comment:
I think this should be defined using the  model metadata extension. We shouldn't need to reference the model directly in the API IIRC, so we hopefully we can remove this.

https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/docs/console-extensions.md#consolemodel-metadata

Reply:
This comment still stands. I don't think we should be creating a K8sModel resource, even if we have to build a reference somewhere.